### PR TITLE
Implemented login and signup buttons

### PR DESCRIPTION
**Pull request by biennviquiera**
Buttons now perform parse login and signup functionality. Alerts appear when an error is encountered and perform the segue to a tableviewcontroller on success. Still need to change the segue to a tab view controller later

### CODE REVIEW

**Review by melissa-gu — APPROVED** [review]
Looks good! Moving forward (wherever applicable) add pixelcloud links to screenshots/videos demoing the new functionality as a sort of "test plan" for the code you're committing

File: Disher/View Controllers/LoginViewController.m [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+- (IBAction)didTapSignup:(id)sender {
+    [self registerUser];
+}
+
+//HELPER METHODS
+
+- (void) loginUser {
+    NSString *username = self.usernameField.text;
+    NSString *password = self.passwordField.text;
```
Comment (ecristerna) [inline Disher/View Controllers/LoginViewController.m:40]: should we perform any checks/validations for these fields? what happens if we send nil to the Parse API?

Comment (ecristerna) [inline Disher/View Controllers/LoginViewController.m:40]: should we perform any checks/validations for these fields? what happens if we send nil to the Parse API?

File: Disher/View Controllers/LoginViewController.m [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    }];
+}
+
+- (void) registerUser {
+    PFUser *newUser = [PFUser user];
+    newUser.username = self.usernameField.text;
+    newUser.password = self.passwordField.text;
+    
+    [newUser signUpInBackgroundWithBlock:^(BOOL succeeded, NSError * _Nullable error) {
+        if (error) {            NSString *errorString = [error userInfo][@"error"];
```
Comment (ecristerna) [inline Disher/View Controllers/LoginViewController.m:68]: break line

File: Disher/View Controllers/LoginViewController.m [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+            NSLog(@"User logged in successfully");
+            [self performSegueWithIdentifier:@"loginSegue" sender:nil];
+        }
+    }];
+}
+
+- (void) registerUser {
+    PFUser *newUser = [PFUser user];
+    newUser.username = self.usernameField.text;
+    newUser.password = self.passwordField.text;
```
Comment (ecristerna) [inline Disher/View Controllers/LoginViewController.m:65]: similar here; what if these fields are actually empty?

File: Disher/View Controllers/LoginViewController.m (hunk 1 above)
Comment (biennviquiera) [reply Disher/View Controllers/LoginViewController.m:40]: I believe the Parse API checks for these conditions. I've added error alerts with the returned error from parse in the completion blocks: 
image

Comment (ecristerna) [reply Disher/View Controllers/LoginViewController.m:40]: 👍

**Pull request merged**